CONTRIBUTING.md

```diff
@@ -99,6 +99,59 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`
 
+### Committing files when updating the `/src` directory
+
+When you run `npm start` to preview the website locally, many files are
+generated that are tracked by Git. This leaves your working directory in a dirty
+state and knowing which files need to be included in your commit can seem like a
```

Comment:
Hey! I'd make the following stylistic changes to the sentence "This leaves ... daunting task" with the following:

"This leaves your working directory in a dirty state, and it can make knowing which files to include in your commit seem like a daunting task."